dietpi/dietpi-software

```diff
@@ -285,6 +285,7 @@ _EOF_
 			'●─ Development & Programming ' #26
 			'●─ Text Editors ' #27
 			'●─ Desktop Utilities ' #28
+			'●- Browsers ' #29
```

Comment:
Please add browsers below "Desktop Utilities" for now. We cause a lot required changes in the docs, including webserver redirects to not break visitors links, when changing categories, so I want to do category changes in dedicated PRs, when required. In the docs, other than here, Chromium is already below "Desktop Utilities", so for that no change is required 😉.

Maybe your plan about how to implement the browser preference implies the category? I think for now we can do it without a dedicated category, based in software ID only, as well. Also Lynx is not a browser for a desktop, so it does not fit well into a browser preference that is followed when selecting a desktop IMO 😉.

Reply:
I implemented Lynx because I thought that it would be best to have another software for the browser preference.  If we keep it, which category would we put it in?

Reply:
Generally it's good to have some more choices, but not a requirement to allow the choice, even when it's only between two options. Since currently I have no further idea than using the browser preference for desktops, all options should be reasonable for desktop usage, where I'm not sure if anyone would use a console-based browser? Although it would be very lightweight, I guess, probably I should try it out :).

Lynx could fit into network tools? But, as long as we indeed do not add it for desktop browser choice, probably it's not worth to keep it, as long as we don't do anything reasonable more than installing the APT package, what you think? At least I was thinking to remove most of the other network tools, probably after adding some documentation troubleshooting/monitoring section where they are used for example. Everyone who knows how to use those tools, definitely knows how to install them the native way via APT.